meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-support/ti-mfgtool-files/ti-mfgtool-files_0.1.bb

```diff
@@ -0,0 +1,48 @@
+SUMMARY = "MFGTOOL Support Files and Binaries"
+LICENSE = "BSD-3-Clause & LGPLv2.1+"
```

Comment:
Why BSD/LGPL while the script itself is MIT based?

Reply:
fixed